frontend/packages/operator-lifecycle-manager/src/components/operator-install-page.tsx

```diff
@@ -0,0 +1,471 @@
+import * as React from 'react';
+import { safeLoad } from 'js-yaml';
+import { Helmet } from 'react-helmet';
+import { match } from 'react-router-dom';
+import { ClusterServiceVersionLogo, iconFor } from './index';
+import {
+  ActionGroup,
+  Alert,
+  Bullseye,
+  Button,
+  Card,
+  CardBody,
+  Spinner,
+} from '@patternfly/react-core';
+import {
+  Firehose,
+  FirehoseResourcesResult,
+  ResourceLink,
+  ResourceStatus,
+  resourcePathFromModel,
+} from '@console/internal/components/utils';
+import {
+  groupVersionFor,
+  k8sPatch,
+  referenceForModel,
+  referenceForGroupVersionKind,
+} from '@console/internal/module/k8s';
+import { errorModal } from '@console/internal/components/modals';
+import {
+  ClusterServiceVersionModel,
+  InstallPlanModel,
+  PackageManifestModel,
+  SubscriptionModel,
+} from '../models';
+import {
+  CRDDescription,
+  ClusterServiceVersionKind,
+  SubscriptionKind,
+  InstallPlanKind,
+} from '../types';
+import { StatusIconAndText } from '@console/shared';
+import {
+  GreenCheckCircleIcon,
+  RedExclamationCircleIcon,
+  YellowExclamationTriangleIcon,
+} from '@console/shared/src/components/status/icons';
+import { InstallPlanPreview } from './install-plan';
+
+const getCRDFromObject = (obj: ClusterServiceVersionKind | InstallPlanKind | SubscriptionKind) => {
+  return obj?.metadata?.annotations?.['operatorframework.io/initialization-resource'];
+};
+
+const ViewInstalledOperatorsButton: React.FC<InstallPageMessageButtonProps> = ({
+  namespace,
+  className,
+}) => (
+  <Button
+    className={className}
+    data-test="view-installed-operators-btn"
+    variant="link"
+    component="a"
```

Comment:
All of these buttons need to use a `Link` instead of an `a` element. Otherwise the entire console reloads.